routes/verify.go

```diff
@@ -452,3 +484,380 @@ func (fes *APIServer) sendEmail(email *mail.SGMailV3) {
 func (fes *APIServer) IsConfiguredForSendgrid() bool {
 	return fes.Config.SendgridApiKey != ""
 }
+
+//
+// JUMIO
+//
+
+func (fes *APIServer) IsConfiguredForJumio() bool {
+	return fes.Config.JumioToken != ""  && fes.Config.JumioSecret != ""
+}
+
+type JumioInitRequest struct {
+	CustomerInternalReference string `json:"customerInternalReference"`
+	UserReference             string `json:"userReference"`
+	SuccessURL                string `json:"successUrl"`
+	ErrorURL                  string `json:"errorUrl"`
+}
+
+type JumioInitResponse struct {
+	RedirectURL          string `json:"redirectUrl"`
+	TransactionReference string `json:"transactionReference"`
+}
+
+type JumioBeginRequest struct {
+	PublicKey string
+	SuccessURL string
+	ErrorURL string
+	JWT       string
+}
+
+type JumioBeginResponse struct {
+	URL string
+}
+
+func (fes *APIServer) JumioBegin(ww http.ResponseWriter, req *http.Request) {
+	decoder := json.NewDecoder(io.LimitReader(req.Body, MaxRequestBodySizeBytes))
+	requestData := JumioBeginRequest{}
+	if err := decoder.Decode(&requestData); err != nil {
+		_AddBadRequestError(ww, fmt.Sprintf("JumioBegin: Problem parsing request body: %v", err))
+		return
+	}
+
+	isValid, err := fes.ValidateJWT(requestData.PublicKey, requestData.JWT)
```

Comment:
Do we need check the err here? I saw you did above but those checks might be unnecessary. Can't imagine a case where isValid would be true and err would present

Reply:
should check error for completeness.